app/assets/javascripts/application.js

```diff
@@ -234,6 +240,7 @@ function template_info(div, url) {
     },
     error: function(jqXHR, textStatus, errorThrown) {
       $(div).html('<div class="alert alert-warning alert-dismissable">' +
+        '<span class="pficon-layered"><span class="pficon pficon-warning-triangle"></span><span class="pficon pficon-warning-exclamation"></span></span>' +
```

Comment:
i think the proxy one will be merged first as its nearly ready,then you could rebase on top of it.